src/Maestro/tests/DependencyUpdateErrorProcessor.Tests/UpdateGithubIssueTest.cs

```diff
@@ -0,0 +1,131 @@
+            Issue issueCreated = GetIssue();
+            List<NewIssue> newIssues = new List<NewIssue>();
+            GithubClient.Setup(x => x.Issue.Create(It.IsAny<long>(), Capture.In(newIssues))).ReturnsAsync(issueCreated);
+            GithubClient.Setup(x => x.Issue.Get(0, 1)).ReturnsAsync(issueCreated);
+            List<IssueUpdate> issueUpdate = new List<IssueUpdate>();
+            GithubClient.Setup(x => x.Issue.Update(0, 1, Capture.In(issueUpdate))).ReturnsAsync(issueCreated);
+            DependencyUpdateErrorProcessor errorProcessor =
+                ActivatorUtilities.CreateInstance<DependencyUpdateErrorProcessor>(Scope.ServiceProvider,
+                    Context);
+            await errorProcessor.ProcessDependencyUpdateErrorsAsync();
+            string issueBody = $@"The following errors have been detected when attempting to update dependencies in 
+'https://github.com/maestro-auth-test/maestro-test2'
+
+ [marker]: <> (subscriptionId: 'ee8cdcfb-ee51-4bf3-55d3-08d79538f94d', method: 'UpdateAssetsAsync', errorMessage: 'build: 14	Unexpected error processing action: Validation Failed')
+**SubscriptionId:** 'ee8cdcfb-ee51-4bf3-55d3-08d79538f94d'
+**Source Repository :**  'Source Repo'
+**Target Repository :**  'Target Repo'
+**Branch Name :**  '38'
+**Error Message :**  'build: 14	Unexpected error processing action: Validation Failed'
+**Method :** 'UpdateAssetsAsync'
+**Action :** 'Creating new issue'
+**Last seen :** '1/1/2200 12:00:00 AM'
+**/FyiHandle :** @epananth";
+            string updatedIssueBody =
+                $@"The following errors have been detected when attempting to update dependencies in
+'https://github.com/maestro-auth-test/maestro-test2'
+
+  [marker]: <> (subscriptionId: 'ee8cdcfb-ee51-4bf3-55d3-08d79538f94d', method: 'UpdateAssetsAsync', errorMessage: 'build: 14	Unexpected error processing action: Validation Failed')
+**SubscriptionId:** 'ee8cdcfb-ee51-4bf3-55d3-08d79538f94d'
+**Source Repository :**  'Source Repo'
+**Target Repository :**  'Target Repo'
+**Branch Name :**  '38'
+**Error Message :**  'build: 14	Unexpected error processing action: Validation Failed'
+**Method :** 'UpdateAssetsAsync'
+**Action :** 'Updating existing issue'
+**Last seen :** '2/1/2200 12:00:00 AM'
+**/FyiHandle :** @epananth";
+            GithubClient.Verify(x => x.Issue.Update(0, 1, It.IsAny<IssueUpdate>()), Times.Once);
+            Assert.Single(newIssues);
+            Assert.Equal("[Dependency Update] Errors during dependency updates to : https://github.com/maestro-auth-test/maestro-test2", newIssues[0].Title);
+            Assert.Equal("DependencyUpdateError", newIssues[0].Labels[0]);
+            Assert.Equal(issueBody, newIssues[0].Body);
+            Assert.Equal(updatedIssueBody, issueUpdate[0].Body);
+        }
+
+        public Issue GetIssue()
+        {
+            Issue issue = new Issue
```

Comment:
There are a lot of parameters here, I'd put parameter names on all of the arguments here so that it's easier to tell what "1" and "12" and all the nulls are supposed to be.